support-frontend/app/controllers/Application.scala

```diff
@@ -310,6 +312,11 @@ class Application(
       ),
     ).withSettingsSurrogateKey
   }
+
+  def eventsRouter(countryGroupId: String, eventId: Option[String]) = authAction { implicit request =>
```

Comment:
Not sure I understand difference between `MaybeAuthenticatedAction` used elsewhere and  `authAction` - presumably both allow us to share settings?

Reply:
`MaybeAuthenticatedAction` authenticates against Okta, so used for readers and people with Guardian Identity accounts.

This `authAction` uses Google auth, so only people with Guardian Google accounts can access this.

Great to point out that it's not massively clear. We need a little work on the identity stuff.